lib/controllers/cats.js

```diff
@@ -1,14 +1,20 @@
 const { Router } = require('express');
-const { cats } = require('../../data/cats');
+const Cat = require('../models/Cat');
 
 module.exports = Router()
-  .get('/:id', (req, res) => {
+  .get('/:id', async (req, res) => {
     const id = req.params.id;
-    const matchCats = cats.find((cat) => cat.id === id);
+    const matchCats = await Cat.getById(id);
     res.json(matchCats);
   })
 
-  .get('/', (req, res) => {
-    const allCats = cats.map((cat) => ({ id: cat.id, name: cat.name }));
-    res.json(allCats);
+  .get('/', async (req, res) => {
+    const Cats = await Cat.getAll();
+    const newCats = Cats.map((cat) => {
```

Comment:
You're doing a little extra work here; since the `getAll()` returns an array we don't need to map over it again. If you really just want to return the id and name you can change your SQL query to be: `SELECT id, name FROM cats` and that will get you an array of the id's and names for that route. Saves you some munging in the controller.